wire-ios/Wire-iOS/Sources/UserInterface/UserProfile/Devices/ProfileDeviceDetailsView.swift

```diff
@@ -0,0 +1,223 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import WireCommonComponents
+
+struct ProfileDeviceDetailsView: View {
+    @Environment(\.dismiss)
+    private var dismiss
+
+    @StateObject var viewModel: DeviceInfoViewModel
```

Comment:
```suggestion
    @ObservedObject: var viewModel: DeviceInfoViewModel
```

> As described in the previous section: if you inject the observed object, you should use @ObservedObject.
https://www.avanderlee.com/swiftui/stateobject-observedobject-differences/

Reply:
done